sample-apps/javascript-sample-app/server.js

```diff
@@ -0,0 +1,123 @@
+/*
```

Comment:
lets try to be consistent. Not all files have the license header. can you make sure that you add in all?